src/server/qgsserverprojectutils.cpp

```diff
@@ -333,6 +333,55 @@ QStringList QgsServerProjectUtils::wmsOutputCrsList( const QgsProject &project )
   return crsList;
 }
 
+QStringList QgsServerProjectUtils::wmsOutputCrsListAsOgcUrn( const QgsProject &project )
```

Comment:
Could we not call `wmsOutputCrsList()`and then transform them to ogcurn ? To avoid code duplication

Reply:
Good idea. Done.